packages/react/src/components/PageHeader/PageHeader.tsx

```diff
@@ -241,6 +235,176 @@ PageHeaderContent.propTypes = {
   pageActions: PropTypes.node,
 };
 
+/**
+ * ----------------
+ * PageHeaderContentPageActions
+ * ----------------
+ */
+interface PageHeaderContentPageActionsProps {
+  /**
+   * Provide child elements to be rendered inside PageHeaderContentPageActions.
+   */
+  children?: React.ReactNode;
+  /**
+   * Specify an optional className to be added to your PageHeaderContentPageActions
+   */
+  className?: string;
+  /**
+   *  The PageHeaderContent's page actions collabpsible Menu button label
+   */
+  menuButtonLabel?: string;
+  /**
+   * The PageHeaderContent's page actions
+   */
+  pageActions?: React.ReactNode;
+}
+const PageHeaderContentPageActions = React.forwardRef<
```

Comment:
Regarding the question in https://github.com/carbon-design-system/carbon/pull/19050#discussion_r2039839176, could this be a regular component since there is no `ref`? This question also applies to other components in this file that don't have a `ref` and use `forwardRef`.